tests/e2e/vLLM/test_vllm.py

```diff
@@ -71,6 +71,7 @@ def set_up(self):
             pytest.skip("Skipping test; cadence mismatch")
 
         self.model = eval_config["model"]
+        self.model_class = eval_config.get("model_class", "AutoModelForCausalLM")
```

Comment:
is this supposed to be a string or a callable?

Reply:
it should be a string, we use `getattr` in the code to map it to the actual class/callable

Reply:
Oh I see, eval_config is a dict.